### PR TITLE
Adding Chatty.jar v0.8.2

### PR DESCRIPTION
**Pull request by cblecker**
- [X] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [X] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [X] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [X] Commit message includes cask’s name.
- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` left no offenses.
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/chatty.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+cask 'chatty' do
+  version '0.8.2'
+  sha256 '32f1b544bb140852f9e2bf6ce0f55bbf50dd5d80880a35005303f45fe266cd91'
+
+  url "https://github.com/chatty/chatty/releases/download/v#{version}/Chatty_#{version}.zip"
+  name 'Chatty'
+  homepage 'https://chatty.github.io'
+  license :mit
+
+  app 'Chatty.jar', target: 'Chatty'
```
Comment (vitorgalvao) [inline Casks/chatty.rb]: [No renaming](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/app.md#target-should-only-be-used-in-select-cases).

File: Casks/chatty.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+cask 'chatty' do
+  version '0.8.2'
+  sha256 '32f1b544bb140852f9e2bf6ce0f55bbf50dd5d80880a35005303f45fe266cd91'
+
+  url "https://github.com/chatty/chatty/releases/download/v#{version}/Chatty_#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/chatty.rb]: Add under

```ruby
appcast 'https://github.com/chatty/chatty/releases.atom',
         checkpoint: '56ed9d68dca565405beb5d0b4e9a5a788fee9a1942adb50ad4127daaf6d55fea'
```

File: Casks/chatty.rb (hunk 1 above)
Comment (cblecker) [inline Casks/chatty.rb]: Is there a preferred way to hide the extension on the alias? The default contains the .jar extension (not automatically hidden in OS X like a .app would be)

Comment (vitorgalvao) [inline Casks/chatty.rb]: There is not.

**Comment by cblecker** [comment]
Added appcast and removed the target rename, per @victorpopkov.

**Pull request merged**